pages/linux/rtcwake.md

```diff
@@ -0,0 +1,36 @@
+# rtcwake
+
+> Enter a system sleep state until specified wakeup time relative to your bios clock.
+> Commands need to be run as superuser since shutdowns are involved.
+
+- Check your hardware clock info and exit:
+
+`rtcwake -m show -v`
+
+- Show if an allarm is set:
+
+`rtcwake -m show`
+
+- Disable a previous set alarm:
+
+`rtc -m disable`
+
+- Suspend to ram and wakeup after 10 seconds:
+
+`# rtcwake -m mem -s {{10}}`
```

Comment:
Is the hash needed here? If it's denoting a superuser command, perhaps you could use `sudo ` instead?

Reply:
It's needed to be superuser, sincerely, I was unsure about using the hash or the sudo.. Gonna switch to the last one =)